test/e2e/apimachinery/garbage_collector.go

```diff
@@ -110,7 +109,7 @@ var (
 	gcInformerResyncRetryTimeout = time.Minute
 
 	// CronJobGroupVersionResource unambiguously identifies a CronJob resource.
-	CronJobGroupVersionResource = schema.GroupVersionResource{Group: batchv1beta1.GroupName, Version: "v1beta1", Resource: "cronjobs"}
+	CronJobGroupVersionResource = schema.GroupVersionResource{Group: batchv1.GroupName, Version: "v1", Resource: "cronjobs"}
```

Comment:
This is not needed at all, you can drop the skip below in the test, since CronJob is now GA and is present in every conformant k8s distribution.